airflow/providers/docker/operators/docker.py

```diff
@@ -189,7 +190,20 @@ def __init__(
     ) -> None:
         super().__init__(**kwargs)
         self.api_version = api_version
-        self.auto_remove = auto_remove
+        if type(auto_remove) == bool:
+            warnings.warn(
+                "bool value is deprecated, please use 'never', 'success', or 'force' instead",
```

Comment:
```suggestion
                "bool value for auto_remove is deprecated, please use 'never', 'success', or 'force' instead",
```

bool is the type so users might not understand the message relates to auto_remove